compiler/rustc_index/src/lib.rs

```diff
@@ -50,3 +50,13 @@ macro_rules! static_assert_size {
         const _: (usize, usize) = ($size, ::std::mem::size_of::<$ty>());
     };
 }
+
+#[macro_export]
+macro_rules! indexvec {
+    ($expr:expr; $n:expr) => {
```

Comment:
We also have `IndexVec::from_elem_n`. Probably doesn't hurt to have this as well, just thought I'd mention it.